CONTRIBUTING.md

```diff
@@ -10,30 +10,36 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 1. Run the following commands:
 
 ```sh
-$ git clone YOUR_YARN_REPO_URL
-$ cd yarn
-$ yarn
-$ yarn run build
+git clone YOUR_YARN_REPO_URL
+cd yarn
+yarn
+yarn run build
 ```
 
 ## Building
 
 ```sh
-$ yarn run build
+yarn run build
 ```
 
 ```sh
-$ yarn run watch
+yarn run watch
+```
+
+## Using the local builds
+
+```sh
+alias yarn="node /path/to/yarn/lib/cli/index.js"
```

Comment:
This is an interesting approach... I usually just use the full path (eg. `c:\src\yarn\bin\yarn`)

I'd suggest using `bin/yarn` rather than `lib/cli/index.js`.